save-plugins/warn-plugin/src/commonTest/kotlin/org/cqfn/save/plugin/warn/WarnPluginTest.kt

```diff
@@ -411,6 +411,53 @@ class WarnPluginTest {
         }
     }
 
+    @Test
```

Comment:
Do we still need such tests, while we already test it in examples?

Reply:
No. We do not have tests with multiline warnings

Reply:
But I add `examples/kotlin-diktat/warn-dir/chapter4/GenericFunctionTest.kt`, that's what I mean

Reply:
I think more tests are better :)

Reply:
I think both places are improper for such tests. This is simply a test for warnings extraction, it shouldn't require a whole new test in examples, and neither a functional-like test in `WarnPluginTest`. We have tests for different kinds of warnings extraction in [`UtilsTest.kt`](https://github.com/diktat-static-analysis/save/blob/main/save-plugins/warn-plugin/src/commonNonJsTest/kotlin/org/cqfn/save/plugin/warn/utils/UtilsTest.kt), there you can add more examples with less new additions, than with `examples` directory.